### DISCUSSION TITLE
Ultrawide UI Rescaler Plugin - Offer Bounty

### DISCUSSION THREAD

**Discussion by mbemis777** [runelite/runelite#15615] [Q&A] [answered]
This plugin would be to allow the movement of forefront UI elements (plugin overlays, minimap, inventory, etc.) across the x-access of the client. 

Playing Runescape in fullscreen on an ultrawide is awesome! except when you actually have to do anything... Moving your head and mouse cursor from one side to the other, just to equip something is neck-breaking. It would be nice to choose where these UI elements lie on the screen. Is this possible? I'd even consider providing a bounty for this plugin depending on the level of effort needed. 
![ultrawideRescaler_25](https://user-images.githubusercontent.com/90067058/188327667-c4143b46-6a09-496e-84d4-3e9801ae1542.png)

**Comment by Enriath** [comment, marked as answer]
Hold Alt and drag them

**Reply by mbemis777** [reply]
You're a beast m8. Love you & Thx
